src/content/1.7/development/page-reference/back-office/order/view-order/_index.md

```diff
@@ -0,0 +1,86 @@
+- **OrderController** [src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php) (other domain controllers are used as well, those will be mentioned in related block references bellow) 
+- **Commands** & **Queries** at [src/Core/Domain/Order](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Core/Domain/Order) in `Command`, `Query` directories.
+- Most **handlers** or **domain services implementations** can be found in `Adapter` namespace [src/Adapter/Order](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Adapter/Order)
+
+Once opened, the page will show following blocks:
+
+- [Actions block]({{< relref "#actions-block" >}})
+- [Customer block]({{< relref "#customer-block" >}})
+- [Products block]({{< relref "#products-block" >}})
+- [Messages block]({{< relref "#messages-block" >}})
+- [History block]({{< relref "#history-block" >}})
+- [Payments block]({{< relref "#payments-block" >}})
+
+#### Actions block
+{{< figure src="./img/actions-block.png" title="Order actions block" >}}
+
+This block contains following actions:
+- Status update - new order status can be selected from a dropdown. See [OrderController::updateStatusAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- View invoice - generates invoice PDF. See [OrderController::generateInvoicePdfAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Print order - prepares and shows modified page layout for printing. This action is performed by javascript in [admin-dev/themes/new-theme/js/pages/order/view](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/admin-dev/themes/new-theme/js/pages/order/view)
+- Refund - read about refunds [here]({{< relref "./refunds" >}}).
+- Next/Previous - allows to quickly jump to next or previous order in the list (simple `href` buttons redirecting to another `View order` page).
+
+#### Customer block
+{{< figure src="./img/customer-block.png" title="Order customer block" >}}
+
+Contains information like customer `email`, `addresses`, `orders count`, `total spent sum`. Click the `View full details` link to see more information about the customer (it redirects to `Customers -> Customers -> View customer` page). This block also allows following actions:
+- Editing and selecting new customer shipping and/or invoice addresses (both of these actions opens a modal with a `Customers -> Addresses -> Edit` page inside an `iframe` using the `Lite display` mode of the Back Office). See [OrderController::changeCustomerAddressAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
```

Comment:
Second time we mention Lite Display.

Would it be possible to submit another PR to explain it 😉  ?
It's a TODO-item for docs https://github.com/PrestaShop/docs/issues/231 😄